src/main/java/ru/svgogin/service/spark/handler/RestResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,36 @@
+package ru.svgogin.service.spark.handler;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+import ru.svgogin.service.spark.errordto.CompanyErrorDto;
+import ru.svgogin.service.spark.exception.EntityAlreadyExistsException;
+import ru.svgogin.service.spark.exception.NoSuchEntityException;
+
+@ControllerAdvice
+public class RestResponseEntityExceptionHandler extends ResponseEntityExceptionHandler {
+  private static final Logger log = LoggerFactory
+      .getLogger(RestResponseEntityExceptionHandler.class);
+
+  @ExceptionHandler(value
+                        = {EntityAlreadyExistsException.class, NoSuchEntityException.class})
```

Comment:
nit: it is better to do it this way:
```suggestion
  @ExceptionHandler(value = {
      EntityAlreadyExistsException.class, NoSuchEntityException.class
  })
```

Reply:
Done